app/src/main/java/dev/arkbuilders/navigator/data/stats/PlainStatsStorage.kt

```diff
@@ -27,7 +27,7 @@ class PlainStatsStorage(
     private val index: RootIndex,
     private val preferences: Preferences,
     private val tagStorage: RootTagsStorage,
-    private val statsFlow: SharedFlow<StatsEvent>
+    statsFlow: SharedFlow<StatsEvent>
```

Comment:
Can it be queried directly now?

Reply:
@kirillt No, because `statsFlow` is now just a parameter, not a property.